src/gtfs/read.rs

```diff
@@ -191,12 +191,12 @@ struct Route {
     route_type: RouteType,
     #[serde(rename = "route_url")]
     url: Option<String>,
-    #[serde(rename = "route_color", default, deserialize_with = "string_to_color",
-            serialize_with = "ser_from_color")]
+    #[serde(rename = "route_color", default, deserialize_with = "string_to_color")]
```

Comment:
I think you don't need a custom deserializer, it should work out of the box.

Reply:
you're right, it's automagic ! :D